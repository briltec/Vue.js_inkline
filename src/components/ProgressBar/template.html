<div class="bar"
     :class="classes['*']"
     :style="style"
     role="progressbar"
     :aria-valuemin="min"
     :aria-valuemax="max"
     :aria-valuenow="progress">
    <slot></slot>
</div>