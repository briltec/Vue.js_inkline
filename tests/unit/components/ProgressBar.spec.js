import { mount } from '@vue/test-utils';
import ProgressBar from 'inkline/components/ProgressBar';

describe('Components', () => {
    describe('ProgressBar', () => {
        let wrapper;

        beforeEach(() => {
            wrapper = mount(ProgressBar);
        });

        test('should render correctly', () => {
            expect(wrapper.html()).toMatchSnapshot();
        });
    });
});
