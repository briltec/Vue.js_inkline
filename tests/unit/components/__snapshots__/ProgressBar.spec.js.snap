// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components ProgressBar should render correctly 1`] = `
<div class="alert">
  <!---->
  <div class="content"></div>
  <!---->
</div>
`;
